script/incremental_build.sh

```diff
@@ -10,10 +10,20 @@ if [ "${BRANCH_NAME}" = "master" ]; then
 else
   # Make sure there is up-to-date master.
   git fetch origin master
-  BRANCH_BASE_SHA=$(git merge-base --fork-point FETCH_HEAD HEAD)
-  echo "Checking changes from $BRANCH_BASE_SHA..."
-  FLUTTER_CHANGED_GLOBAL=`git diff --name-only $BRANCH_BASE_SHA HEAD | grep -v packages | wc -l`
-  FLUTTER_CHANGED_PACKAGES=`git diff --name-only $BRANCH_BASE_SHA HEAD | grep -o "packages/[^/]*" | sed -e "s/packages\///g" | sort | uniq | paste -s -d, -`
+
+  FLUTTER_CHANGED_GLOBAL=0
+  FLUTTER_CHANGED_GLOBAL=""
```

Comment:
Is it intentional to first assign 0 and then empty string?

Reply:
Good catch! It's a typo. It suppose to be `FLUTTER_CHANGED_PACKAGES`. Updated the PR.